Templates/Android/generated/BaseEntityCollectionReferenceRequest.java.tt

```diff
@@ -39,37 +39,43 @@
                 .post(new<#=TypeName(c)#>, body);
     }
 
+<# if (c.GetFeatures().CanExpand) { #>
     /**
      * Sets the expand clause for the request
      *
      * @param value The expand clause
      * @return The updated request
      */
     public <#=ITypeCollectionReferenceRequest(c)#> expand(final String value) {
-        addQueryOption(new QueryOption("expand", value));
+        addQueryOption(new QueryOption("$expand", value));
         return (<#=TypeCollectionReferenceRequest(c)#>)this;
     }
 
+<# } #>
+<# if (c.GetFeatures().CanSelect) { #>
```

Comment:
Per conversation with @MIchaelMainer @shiftylogic we're leaving the current implementation as-is